pycc/ccwfn.py

```diff
@@ -299,6 +299,11 @@ def build_Wmnij(self, o, v, ERI, t1, t2):
         if self.model == 'CCD':
             Wmnij = ERI[o,o,o,o].copy()
             Wmnij = Wmnij + contract('ijef,mnef->mnij', t2, ERI[o,o,v,v])
+        elif self.model == 'CC2':
+            Wmnij = ERI[o,o,o,o].copy()
+            Wmnij = Wmnij + contract('je,mnie->mnij', t1, ERI[o,o,o,v])
+            Wmnij = Wmnij + contract('ie,mnej->mnij', t1, ERI[o,o,v,o])
+            Wmnij = Wmnij + contract('ijef,mnef->mnij', self.build_tau(t1, t2, fact1=0), ERI[o,o,v,v])
```

Comment:
Isn't the default `fact1=1.0`?  If so, is this block different from the CCSD block below it?

Reply:
I mis-read this in every single instance!  That `1` after `fact` messes with my eyes!